src/Orleans/Configuration/GlobalConfiguration.cs

```diff
@@ -593,6 +599,8 @@ internal GlobalConfiguration()
 
             ProviderConfigurations = new Dictionary<string, ProviderCategoryConfiguration>();
             GrainServiceConfigurations = new GrainServiceConfigurations();
+            DefaultVersionCompatibilityStrategy = BackwardCompatible.Singleton;
+			DefaultVersionPlacementStrategy = MinimumVersionPlacement.Singleton;
```

Comment:
Yeah I think my VS2017 is set to use tabs instead of spaces... will fix that, sorry